pkg/catalog/manager/manager.go

```diff
@@ -226,6 +226,10 @@ func (m *Manager) ValidateKubeVersion(template *v3.CatalogTemplateVersion, clust
 		return err
 	}
```

Comment:
And `cluster` would never be nil because you'd get a NotFound error above right ?

Reply:
correct